tests/test_filtering.py

```diff
@@ -164,6 +164,15 @@ async def test_filter_not_with_or(self):
         self.assertEqual(len(tournaments), 2)
         self.assertSetEqual({t.name for t in tournaments}, {"0", "1"})
 
+    async def test_filter_date(self):
+        await DatetimeFields.create(
+            datetime=datetime.datetime(
+                year=2020, month=5, day=20, hour=0, minute=0, second=0, microsecond=0
+            )
+        )
+        date = datetime.date(2020, 5, 20)
+        self.assertEqual(await DatetimeFields.filter(datetime__date=date).count(), 1)
```

Comment:
We should have more tests for these: checking that this filter modifier actually converts a date time to a date. The provided test case doesn't actually test anything. For instance, in Postgres

```
SELECT TIMESTAMP '2024-01-20 00:00:00' = DATE '2024-01-20';
> true
```